controllers/feeds/getOneFeedChat.js

```diff
@@ -134,14 +138,16 @@ const getOneFeedChatService = async (req, res) => {
       },
       expired_at: data.results[0].expired_at,
       totalComment: data.results[0].reaction_counts?.comment ?? 0,
-      isOwnPost: await checkIsOwnPost(req.userId, data.results[0].actor.id),
+      isOwnPost: req.userId === postMakerUserId,
+      isOwnSignedPost: selfSignedUserId === postMakerUserId,
+      isOwnAnonymousPost: selfAnonymousUserId === postMakerUserId,
       totalCommentBadge: comments.filter((comment) => comment.actor.id !== req.userId).length ?? 0,
       type: 'post-notif',
       titlePost: data.results[0].object?.message ?? data.results[0].message,
       downvote: data.results[0].reaction_counts?.downvotes ?? 0,
       upvote: data.results[0].reaction_counts?.upvotes ?? 0,
       postMaker: actor,
-      isAnonym: data.results[0].object?.anonimity ?? data.results[0].anonimity,
+      isAnonym,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for determining `isOwnPost`, `isOwnSignedPost`, and `isOwnAnonymousPost` seems to be incorrect. You are comparing `req.userId` with `postMakerUserId` for `isOwnPost`, but for `isOwnSignedPost` and `isOwnAnonymousPost`, you are comparing `selfSignedUserId` and `selfAnonymousUserId` respectively with `postMakerUserId`. This could lead to inconsistent results if `req.userId` is not the same as `selfSignedUserId` or `selfAnonymousUserId`. Please review this logic.
```diff
      isOwnPost: req.userId === postMakerUserId,
      isOwnSignedPost: selfSignedUserId === postMakerUserId,
      isOwnAnonymousPost: selfAnonymousUserId === postMakerUserId,
       totalCommentBadge: comments.filter((comment) => comment.actor.id !== req.userId).length ?? 0,
       type: 'post-notif',
       titlePost: data.results[0].object?.message ?? data.results[0].message,
       downvote: data.results[0].reaction_counts?.downvotes ?? 0,
       upvote: data.results[0].reaction_counts?.upvotes ?? 0,
       postMaker: actor,
      isAnonym,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->